Sources/StreamChat/WebSocketClient/Events/MessageEvents.swift

```diff
@@ -241,3 +246,75 @@ public struct NewMessageErrorEvent: Event {
     public let messageId: MessageId
     public let error: Error
 }
+
+// MARK: - Workaround to map a deleted message to Model.
+
+// At the moment our SDK does not support mapping Payload -> Model
+// So this is just a workaround for `MessageDeletedEvent` to have the `message` non-optional.
+// So some of the data will be incorrect, but for this is use case is more than enough.
```

Comment:
which data would be incorrect? Isn't this everything coming from the payload?

Reply:
No, we are not mapping reactions, the `isSentByCurrentUser` is also a computed property on ou DTOs, so that is also incorrect. `isFlaggedByCurrentUser` as well... etc.. But those are minors